src/fsharp/FSharp.Core/map.fs

```diff
@@ -64,10 +55,10 @@ namespace Microsoft.FSharp.Collections
 
         let empty = MapEmpty 
 
-        let height  = function
+        let size  = function
```

Comment:
please inline size function

Reply:
No really keen to inline whilst MapOne exists. `size` as it stands has to do two attempted types casts which easily outweight a function call.